docs/source/getting_started/installation.rst

```diff
@@ -220,30 +193,29 @@ Docker system.
 Step 9: Check Project Setup Works
 =================================================
 
-To check the project works, open a terminal in the project root directory,
-which is the ``cs-field-guide/`` directory (should contain a file called
-``csfg``).
+To check the project works, open a terminal in the project root directory, which is the ``cs-field-guide/`` directory (should contain a file called ``csfg``).
 
-Type the following command into the terminal (we will cover this command
-in more detail on the next page):
+Type the following command into the terminal (we cover this command in more detail on the helper commands page):
 
 .. code-block:: bash
 
     $ ./csfg start
 
-If this is the first time you're running this script, it will need to build
-system images.
-This can take some time, roughly 15 to 30 minutes, depending on your computer
-and internet speed (we recommend grabbing a cup of tea and watching an episode
-of Brooklyn Nine-Nine on Netflix).
+If this is the first time you're running this script it will need to build system images.
+This may take roughly 15 to 30 minutes, depending on your computer and internet speed.
 
-After the helper script builds the system images, it will automatically start
-the system, and will let you know when the system is ready.
-You should then be able to open your preferred web browser to the URL
-``localhost`` and see the CS Field Guide homepage.
+Once the `./csfg start` command has completed, you will need to run the update command.
```

Comment:
Use the triple backtick method here

Reply:
Howcome? `./csfg start` isn't really a code _block_

Reply:
A) it's more consistent:
![image](https://user-images.githubusercontent.com/25916475/59389598-c7fe1480-8dc2-11e9-9275-eb9f42af5412.png)

B) It looks more like italics as is than code
![image](https://user-images.githubusercontent.com/25916475/59389642-e6641000-8dc2-11e9-89b0-db53332714b3.png)

Just firefox things I suppose

Reply:
Is it meant to be double backticks? Wither way, not single ones